src/NzbDrone.Core/Datastore/Migration/158_add_indexer_tags.cs

```diff
@@ -0,0 +1,14 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(158)]
```

Comment:
I accidentally used migration 158 already before checking existing PRs. My apologies. Can you bump this to 159?

PS: If needed, you can edit your existing db and renumber the applied migration to 159 in table VersionInfo table without having to restore from backup.

Reply:
Migration ID updated.